modules/control-disablements/main.tf

```diff
@@ -0,0 +1,61 @@
+#-----------------------------------------------------------------------------------------------------------------------
+# DATA SOURCES AND LOCALS
+#-----------------------------------------------------------------------------------------------------------------------
+data "aws_caller_identity" "this" {}
+data "aws_region" "this" {}
+
+locals {
+  is_global_resource_region     = data.aws_region.this.name == var.global_resource_collector_region
+  is_central_cloudtrail_account = data.aws_caller_identity.this.account_id == var.central_logging_account
+
+  all_global_region_controls = [
+    "arn:aws:securityhub:${data.aws_region.this.name}:${data.aws_caller_identity.this.account_id}:control/aws-foundational-security-best-practices/v/1.0.0/Config.1",
```

Comment:
We should use the AWS partition here no?

Reply:
12d28d1b9a60a8c590d8074c6983d4358624c3b4